tslint.json

```diff
@@ -91,7 +91,8 @@
     "radix": true,
     "semicolon": [
       true,
-      "always"
+      "always",
+      "ignore-bound-class-methods"
```

Comment:
My IDE has some troubles without this rule because he always add `;` at the end of arrow function (what is normal) but eslint doesn't accept that.

```
const x = () => {

};
```